python/plugins/processing/algs/gdal/ogr2ogrtopostgislist.py

```diff
@@ -244,7 +255,7 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         arguments.append('-f')
         arguments.append('PostgreSQL')
         arguments.append('PG:"')
-        for token in uri.connectionInfo(executing).split(' '):
+        for token in QgsDataSourceUri(uri).connectionInfo(executing).split(' '):
```

Comment:
Shouldn't use `decodeUri` here?

Reply:
Probably, but I'm reluctant to change anything I don't have to here :smiley: